ansible/playbooks/other.yml

```diff
@@ -0,0 +1,13 @@
+# hosts that Jenkins is going to run playbooks against
+- hosts: other
+  become: true
+  tasks:
+    - name: Create Jenkins user
+      user:
+        name: "{{ jenkins_ssh_user }}"
+        group: wheel
+    - name: Set up SSH key for Jenkins
+      authorized_key:
+        user: "{{ jenkins_ssh_user }}"
+        key: "{{ jenkins_ssh_public_key_data }}"
+    # ...other host setup tasks...
```

Comment:
Is this file just there as an example?

Reply:
Primarily, yes - as this repo is intended to provision other ec2 resources, not just jenkins, so this playbook would make sure the jenkins user exists on the other instances. We just need to decide on the architecture on whether or not we want to do that.